core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/HadoopConfigurationService.java

```diff
@@ -11,6 +15,8 @@
 
 import org.springframework.stereotype.Service;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.sequenceiq.cloudbreak.domain.HostGroup;
 import com.sequenceiq.cloudbreak.domain.Stack;
```

Comment:
Bascially the whole service shall not depend on stack, just cluster. It would be even better if we not even depend on cluster just: bp and hostgroup